sdk/eventhub/event-hubs/samples/runsamples.ts

```diff
@@ -0,0 +1,23 @@
+import * as dotenv from "dotenv";
+dotenv.config();
```

Comment:
what's the behavior of this package if a .env file doesn't exist? Will we have one in CI to make it run anyway?

Reply:
It basically just no-ops. The nice thing is that, aside from this little line, all the rest of the code thinks it's just extracting the values from the environment.

So for our daily dev we can just fill in the .env file and for CI it can just set the values in the environment.